packages/cli/src/commands/org/config/list.ts

```diff
@@ -0,0 +1,76 @@
+import {Command, flags} from '@oclif/command';
+import {AuthenticatedClient} from '../../../lib/platform/authenticatedClient';
+import {getTargetOrg} from '../../../lib/snapshot/snapshotCommon';
+import {Config} from '../../../lib/config/config';
+import {cli} from 'cli-ux';
+import {
+  IsAuthenticated,
+  Preconditions,
+} from '../../../lib/decorators/preconditions';
+import {
+  buildAnalyticsFailureHook,
+  buildAnalyticsSuccessHook,
+} from '../../../hooks/analytics/analytics';
+import dedent from 'ts-dedent';
+
+export default class List extends Command {
+  public static description = 'List available snapshots from an organization';
+
+  public static flags = {
+    ...cli.table.flags(),
+    target: flags.string({
+      char: 't',
+      description:
+        'The unique identifier of the organization containing the snapshots. If not specified, the organization you are connected to will be used.',
+      helpValue: 'destinationorganizationg7dg3gd',
+      required: false,
+    }),
+  };
+
+  public static args = [{name: 'file'}];
+
+  @Preconditions(IsAuthenticated())
+  public async run() {
+    const {flags} = this.parse(List);
+    const org = await getTargetOrg(this.configuration, flags.target);
+    const platformClient = await new AuthenticatedClient().getClient({
+      organization: org,
+    });
+    const snapshots = await platformClient.resourceSnapshot.list();
+    if (snapshots.length === 0) {
+      this.log(
+        dedent(`
+      There is no configuration snapshot available in organization ${org}
+      You can create one using org:config:pull
```

Comment:
I don't think this line is needed since the `pull` command will delete the snapshot once its content is added to the user's workspace.